eradiate/scenes/measure/_core.py

```diff
@@ -97,11 +98,10 @@ def postprocess_results(self, runner_results):
 
         return np.reshape(result, (result.shape[0], result.shape[1]))
 
-    def sensor_infos(self):
+    def sensor_infos(self) -> List[SensorInfo]:
```

Comment:
How do you use this return type hint? I mean, what do you do with it?

Reply:
Ooops! I added this as I was experimenting with Sphinx and forgot to revert this file. Fixing that.

On a more general note: type hints are anyway purely optional, but we could use them for the following:

- Static type checking with mypy: this would allow us to remove some runtime type checking, which is always more pythonic.
- Strict runtime type checking with attrs: attrs fills automatically attributes' `type` field based on type hints. Combined with [attrs-strict](https://github.com/bloomberg/attrs-strict), this allows for boilerplate-free runtime type checking. Example:

  ```python
  import attr
  from attrs_strict import type_validator

  @attr.s
  class MyClass:
      foo: float = attr.ib(validator=type_validator())
  ```
  `foo` will be runtime type-checked with very little code. This, however, should be considered carefully: mypy should generally allow to avoid such things.
- Automated type information in docs: a Sphinx extension can fill in automatically the expected type in the docs. It's however incompatible with our current docstring format; but given how better our Sphinx theme has become, we can probably consider in the future switching to a standard docstring format. This would reduce the chance of desyncing code and docstrings.